agent/tool-scripts/postprocess/pidstat-postprocess

```diff
@@ -63,29 +44,9 @@ my @curr_stats;
 my $interval; # sampling interval
 my $threads;
 
-# When producing CDM docs, this is typically called from the pbench
-# controller only, and the following arguments must be provided:
-if (scalar @ARGV > 2) {
-	#printf "num args: %d args: %s\n", scalar @ARGV, join(" ", @ARGV);
-	die "Must provide <path-to-period.doc> <es-dir> <tool-hostname>" if (scalar @ARGV != 3);
-	$cdm = 1;
-	$period_doc_path = shift;
-	$es_dir = shift;
-	$hostname = shift;
-	my $pidstat_cmd_file = $dir . "/pidstat.cmd";
-	open (FH, $pidstat_cmd_file) || die "Could not open $pidstat_cmd_file";
-	while ($line = <FH>) {
-		#LANG=C /usr/local/bin/pidstat -l -H -w -u -h -d -r  -p ALL   1
-		if ($line =~ /-p\sALL\s+(\d+)/ ){
-			$interval = $1;
-		}
-	}
-	close FH;
-}
-
 my %timestamps;
 my @attributes = get_pidstat_attributes();
-if ($cdm and not defined $interval) {
+if (not defined $interval) {
 	print "Could not determine the sampling interval for pidstat\n";
 	exit 1;
 }
```

Comment:
After this change, in effect `$cmd` will always be false, so lines 50-51 should never be excuted, right?  So, we should be deleting all of 49-52 instead of just the `$cdm` test.